niceman/interface/diff.py

```diff
@@ -104,48 +117,48 @@ def __call__(prov1, prov2):
-
-            repos_1_only = repos_1_s - repos_2_s
-            repos_2_only = repos_2_s - repos_1_s
-
-            if repos_1_only or repos_2_only:
-                print('%s repositories:' % repo_type)
-            if repos_1_only:
-                for repo_id in repos_1_only:
-                    repo = repos_1[repo_id]
-                    print('< %s (%s)' % (repo.identifier, repo.path))
-            if repos_1_only and repos_2_only:
-                print('---')
-            if repos_2_only:
-                for repo_id in repos_2_only:
-                    repo = repos_2[repo_id]
-                    print('> %s (%s)' % (repo.identifier, repo.path))
-
-            for repo_id in repos_1_s.intersection(repos_2_s):
-                repo_1 = repos_1[repo_id]
-                repo_2 = repos_2[repo_id]
-                if repo_1.commit == repo_2.commit:
-                    continue
-                print('%s repository %s:' % (repo_type, repo_id))
-                print('< %s (%s)' % (repo_1.commit, repo_1.path))
-                print('---')
-                print('> %s (%s)' % (repo_2.commit, repo_2.path))
+        # for (dist_type, repo_type) in ((GitDistribution, 'Git repository'),
+        #                                (SVNDistribution, 'SVN repository')):
+        #
+        #     dist_1 = env_1.get_distribution(dist_type)
+        #     if dist_1:
+        #         pkgs_1 = { p._cmp_id: p for p in dist_1.packages }
+        #     else:
+        #         pkgs_1 = {}
+        #     dist_2 = env_2.get_distribution(dist_type)
+        #     if dist_2:
+        #         pkgs_2 = { p._cmp_id: p for p in dist_2.packages }
+        #     else:
+        #         pkgs_2 = {}
+        #
+        #     pkgs_1_s = set(pkgs_1)
+        #     pkgs_2_s = set(pkgs_2)
+        #
+        #     pkgs_only_1 = pkgs_1_s - pkgs_2_s
+        #     pkgs_only_2 = pkgs_2_s - pkgs_1_s
+        #
+        #     if pkgs_only_1 or pkgs_only_2:
+        #         print('%s pkgsitories:' % repo_type)
+        #     if pkgs_only_1:
+        #         for repo_id in pkgs_only_1:
+        #             repo = pkgs_1[repo_id]
+        #             print('< %s (%s)' % (repo.identifier, repo.path))
+        #     if pkgs_only_1 and pkgs_only_2:
+        #         print('---')
+        #     if pkgs_only_2:
+        #         for repo_id in pkgs_only_2:
+        #             repo = pkgs_2[repo_id]
+        #             print('> %s (%s)' % (repo.identifier, repo.path))
+        #
+        #     for repo_id in pkgs_1_s.intersection(pkgs_2_s):
+        #         repo_1 = pkgs_1[repo_id]
+        #         repo_2 = pkgs_2[repo_id]
+        #         if repo_1.commit == repo_2.commit:
+        #             continue
+        #         print('%s pkgsitory %s:' % (repo_type, repo_id))
+        #         print('< %s (%s)' % (repo_1.commit, repo_1.path))
+        #         print('---')
+        #         print('> %s (%s)' % (repo_2.commit, repo_2.path))
```

Comment:
Is there any reason to leave this code commented out rather than deleting it?

Reply:
Probably left over to refer to during refactoring.  Gone now.